### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://hosted.weblate.org/projects/calyxos/seedvault/) for CalyxOS/Seedvault.

Current translation status:

Weblate translation status

### CODE REVIEW

**Review by stevesoltys — APPROVED** [review]

**Review by grote — CHANGES_REQUESTED** [review]
This PR is trivial, but generally, we should merge new translations only after Android 11 is in master which has stricter CI and catches errors.

Also, maybe instead letting weblate open these PRs all the time, maybe we only do it before releases?

**Comment by chirayudesai** [comment]
> This PR is trivial, but generally, we should merge new translations only after Android 11 is in master which has stricter CI and catches errors.

Agreed.
> 
> Also, maybe instead letting weblate open these PRs all the time, maybe we only do it before releases?

I don't see any settings for PR frequency, but what we can do is just keep this open. It'll automatically update / rebase, and then we can merge it:

* When we want to, say after android 11 here
* On some schedule, say bi-weekly or monthly
* Before tagging a release
* Anything else?

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -56,7 +56,7 @@
     <string name="settings_backup_apk_dialog_cancel">Annuler</string>
     <string name="settings_backup_apk_title">Sauvegarde de l’appli</string>
     <string name="settings_category_app_data_backup">Sauvegarde des données de l’appli</string>
-    <string name="settings_auto_restore_summary_usb">Remarque : votre %1$ doit être branché pour que ceci fonctionne.</string>
+    <string name="settings_auto_restore_summary_usb">Remarque : votre %1$s doit être branché pour que ceci fonctionne.</string>
```
Comment (chirayudesai) [inline app/src/main/res/values-fr/strings.xml:59]: @grote look!

I fixed this one manually to verify the check - it wouldn't let me save the wrong version.
Not sure why it's showing a difference in the space at the front though, I was using that to test the checks but it should've been the same again.

Comment (grote) [reply app/src/main/res/values-fr/strings.xml:59]: Great!

Maybe the French have different white-spaces? :P

Comment (mikeNG) [reply app/src/main/res/values-fr/strings.xml:59]: Looks like they used mixed normal space and NO-BREAK SPACE in this file

**Comment by chirayudesai** [comment]
@mikeNG I guess this should target android10 now?

**Comment by chirayudesai** [comment]
We've switched to the stable 'android10' branch for translations, so this is closed in favor of https://github.com/stevesoltys/seedvault/pull/148

**Pull request closed without merging**